### DISCUSSION TITLE
Nuxt Dynamic Components

### DISCUSSION THREAD

**Discussion by hoking1234** [primefaces/community#2426] [PrimeVue]
Referring to https://nuxt.com/docs/guide/directory-structure/components#dynamic-components

My trial:
Fresh install as of https://github.com/primefaces/primevue-examples/tree/main/nuxt-quickstart
Autoimport enabled.

Works fine ✅ when
<InputText v-model="text" placeholder="Type ..." />

Works fine ✅  when
import InputText from 'primevue/inputtext'
<component :is="InputText" v-model="text" placeholder="something..."/>
*** but it seems abit tedious to import for every available component

Nuxt documentation on Dynamic component did give an example for global import for dynamic component as below:
export default defineNuxtConfig({
    components: {
      global: true,
      dirs: ['~/components']
    },
  })

but i cant get it to work as no dir / path is working for Primevue components.

Did I misunderstood something here? 
Can anyone provide an example on how to do dynamic component with Primevue on Nuxt?
Please enlighten me. Thanks

**Comment by j0rgedev** [comment]
Hey @hoking1234, why do you want to use the global import if Nuxt says it's not the recommended way? I haven't used dynamic components in Nuxt since I haven't got that use case, however, I think these dynamic components are meant to be used within your components, so that's why Nuxt gives an example with dirs: ['~/components'], where all your components are placed. Please, share your use case
